docs/product/ai-in-sentry/ai-code-review/index.mdx

```diff
@@ -38,6 +52,39 @@ Once you have added a comment, the assistant will reply, acknowledging the reque
 
 ![Sentry Prevent AI Overview =900x](./img/prevent-ai-response.png)
 
+## GitHub Status Checks
+
+AI Code Review creates GitHub status checks (also called "checks") on your pull requests to indicate whether the code review passed or found potential issues. These status checks appear in the PR's checks section and can be integrated with GitHub's branch protection rules.
+
+### Status Check Behavior
+
+- **Pass**: If AI Code Review doesn't find any errors or concerns in your code, the status check will pass (green checkmark).
+- **Neutral**: If AI Code Review runs successfully but has suggestions or comments, the status check will be neutral (gray) and won't block merging.
+- **Fail**: The status check only fails if there's an error running the AI Code Review itself (for example, if the feature is not enabled), not based on code quality.
+
+### Visibility
+
+Status checks from AI Code Review are visible to:
+- All users with read access to the repository
+- Anyone viewing the pull request on GitHub
+- GitHub Actions and other CI/CD tools that check PR status
+
+The detailed code review comments are also visible to all users with repository access, appearing as regular PR comments.
```

Comment:
hmmmm being a little pedantic here they are "review comments", not "regular PR comments" which are "issue comments", but I don't think it makes much difference for the end user